adapter.js

```diff
@@ -254,16 +292,20 @@ class FlicAdapter extends Adapter {
     }
 
     unload() {
-        if(this.client) {
+        if (this.client) {
+            console.log('Disconnecting client');
             this.client.close();
         }
-        if(this.flicd) {
+
+        if (this.flicd) {
+            console.log('Killing flicd');
             this.flicd.kill();
```

Comment:
This `kill()` call doesn't seem to be working, but I can't figure out why. If you update the config through the Settings UI, it'll restart the adapter and you can see what I mean.